tools/bqtag/bqtag/bqtag.py

```diff
@@ -0,0 +1,800 @@
+"""
+ !/usr/bin/env python
+ -*- coding: utf-8 -*-
+
+ Copyright 2019 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging
+import sys
+import json
+import copy
+import traceback
+import io
+
+from google.cloud import bigquery, datacatalog_v1
+from google.api_core.exceptions import MethodNotImplemented, NotFound
+import google.auth
+
+__all__ = ["BQTableView"]
+
+#############################################################################
+# Create Logger
+#############################################################################
+
+FORMAT = "%(asctime)-15s %(levelname)s %(message)s"
+
+def get_logger(name, fmt):
+    """
+    Creates a Logger that logs to stdout
+    :param name: name of the logger
+    :param fmt: format string for log messages
+    :return: Logger
+    """
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setLevel(logging.DEBUG)
+    handler.setFormatter(logging.Formatter(fmt))
+    log = logging.getLogger(name)
+    log.setLevel(logging.DEBUG)
+    log.addHandler(handler)
+    return log
+
+LOGGER = get_logger("bqtag", FORMAT)
+
+
+#############################################################################
+# Classes to create tree structure to create SQL query for nested columns
+#############################################################################
+
+
+class ColumnNode:
+
```

Comment:
Remove blank line between class declaration and docstring

Reply:
Changed to :

```
class ColumnNode:
    """
    Class to represent a BQ column in a tree
    """
```